docs/products/kafka/howto/connect-with-python.rst

```diff
@@ -0,0 +1,177 @@
+Connect to Aiven for Apache Kafka® with Python
+==============================================
+
+.. See https://docs.google.com/document/d/1PqNbBMzeED_AFeIFzfdcoR_P2bUJrNdcv4uHheeCh6I/edit
+   for an explanation of the following
+
+.. |kafka-python| replace:: ``kafka-python``
+.. _`kafka-python`: https://pypi.org/project/kafka-python/
+
+These examples show how to connect to an Aiven for Apache Kafka® service using the |kafka-python|_ library, as either a producer or consumer.
+
+.. note:: The examples given here provide different options for the different authentication
+   methods. For more information on the supported methods, see `our article on Kafka
+   authentication types <https://developer.aiven.io/docs/products/kafka/concepts/auth-types>`_.
+
+Pre-requisites
+--------------
+
+Install the Python |kafka-python|_ library:
+
+.. code:: bash
+
+    pip install kafka-python
+
+Go to the *Overview* page of your Aiven for Apache Kafka service.
+
+* If you are going to connect with SSL authentication:
+
+  * In the *Connection information* section:
+
+    #. If **Authentication Method** is shown, choose **Client Certificate**
+    #. Next to *Access Key*, click **Download** and save the ``service.key`` file.
+    #. Next to *Access Certificate*, click **Download** and save the ``service.cert`` file.
+    #. Next to *CA Certificate*, click **Download** and save the ``ca.pem`` file.
+
```

Comment:
Can we add a tip about obtaining the same with the cli https://developer.aiven.io/docs/tools/cli/service/user.html#avn-service-user-creds-download ?

Reply:
Oh yes, good point. Thanks.